CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Visit https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+# for learning about the syntax
+
+# Dot files
+.*          @armenzg
+docker/*    @armenzg
```

Comment:
Just needs a Linefeed at the end of the file.  Looks great!  You could add me for review on `ui/push-health` too, if you like.  :)